### ISSUE TITLE
url without trailing / are considered not certified

### ISSUE THREAD

**Issue by Pierre-RA** [healthonnet/is-honcode-certified#1]
Example:
```
$ is-honcode-certified -u https://www.provisu.ch
👻   Be careful
$ is-honcode-certified -u https://www.provisu.ch/
🦄   It's safe
```

**Comment by williambelle** [comment]
See https://github.com/healthonnet/honcode-certification-utils/commit/58946f8a29ba9970e4379e120ee50a402f5737c4

**Comment by williambelle** [comment]
Fixed in #5
